platform/o.n.bootstrap/src/org/netbeans/ModuleManager.java

```diff
@@ -896,7 +896,7 @@ void refineDependencies(Module m, Set<Dependency> dependencies) {
         return installer.refineProvides (m);
     }
     /** Used by Module to communicate with the ModuleInstaller re. classloader. */
-    public ClassLoader refineClassLoader(Module m, List parents) {
+    public ClassLoader refineClassLoader(Module m, List<ClassLoader> parents) {
```

Comment:
Would `public ClassLoader refineClassLoader(Module m, List<? extends ClassLoader> parents)` make more sense? I have really no idea what this does (it is not used in the NB codebase) so it is difficult to judge.

Reply:
From what I understand it enabled a ModuleInstaller to "refine" the classpath of a module. Default Implementation is a no-op.
Searching the codebase however does not find any usage of this method. However the code give a bugzille reference for https://bz.apache.org/netbeans/show_bug.cgi?id=27853.

I foresee no problem changing the type of the list to `List<? extends ClassLoader>`

Reply:
Done